TheMovieDB/TheMovieDB/Controller/ViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  ViewController.swift
+//  TheMovieDB
+//
+//  Created by Jaime Laino on 8/2/21.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    @IBOutlet weak var tableView: UITableView!
+    var movies: [Movie]?
+    lazy var manager: Manager = {
+        return NetworkManager()
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        Facade.shared.retrieveData {[weak self] (result) in
+            switch result {
+            case .success(let movies):
+                self?.movies = movies?.results
+                DispatchQueue.main.async {
+                    self?.tableView.isHidden = false
+                    self?.tableView.reloadData()
+                }
+            case .failure(_):
+                DispatchQueue.main.async {
+                    self?.tableView.isHidden = true
+                }
+            }
+        }
+    }
+    
+}
+
+extension ViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return movies?.count ?? 0
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: "movieCell") as? MovieTableViewCell else {
+            return UITableViewCell()
+        }
+        let movie = movies?[indexPath.row]
+        cell.movieTitle.text = movie?.title ?? ""
```

Comment:
for the labels you don't have to explicitly send a string, if you assign nil to the text it automatically puts an empty string in the label